lib/tasks/translation.rake

```diff
@@ -89,6 +89,7 @@ namespace :translation do
     require 'i18n/tasks/cli'
 
     locale ? I18n::Tasks::CLI.start(["add-missing", "--nil-value", "-l", locale]) : I18n::Tasks::CLI.start(["add-missing", "--nil-value"])
+    RailsTranslationManager::Cleaner.new(Rails.root.join("config", "locales")).clean
```

Comment:
You've tested the `RailsTranslationManager::Cleaner` behaviour, which is great 👍 I think ideally we'd also have a test that calling these rake tasks triggers a call to `RailsTranslationManager::Cleaner`. But I don't think there are any existing tests for the Rake tasks, so this feels a bit out of scope.

Reply:
@ChrisBAshton should we create a new card to write those specs?

Reply:
We wouldn't normally, as it would never get prioritised. It's more something I would hope someone would come in and add at the same time as adding a feature.

What I was trying to say (badly!) was that it would be great if you could add some tests for the rake tasks themselves, because there's a small gap in coverage here - but you don't need to, as there's not yet a precedent in this repo for testing rake tasks. So if you fancy the challenge, go for it, otherwise feel free to ignore my comment 😄

Reply:
@ChrisBAshton Makes sense, I will go ahead and add them!